README.md

```diff
@@ -34,6 +34,16 @@ testing towards [tianocore](https://www.tianocore.org/) utilizing
 * binutils (using `objcopy`)
 * Go >= 1.16
 
+## Available sbctl Packages
+
+List of existing sbctl packages can be found
+[here](https://repology.org/project/sbctl/versions).
+
+In addition, sbctl is also available for [openSUSE &
+Ubuntu](https://software.opensuse.org/package/sbctl?search_term=sbctl). Follow
+the `Expert Download` links to find installation instructions according to your
+operating system.
```

Comment:
Should probably mention that Ubuntu is unofficial and openSUSE is experimental.

Reply:
What is the difference between unofficial and experimental?

Reply:
Well, to me it looks like the openSUSE package is kind of in a "testing" repository, while Ubuntu package is maintained by the community.